ofcourse/bindata.go

```diff
@@ -327,15 +327,15 @@ func AssetNames() []string {
 
```

Comment:
Ah, I might have made an error. But in my understanding, you will have in any case to rebase this branch and regenerate the bindata, correct? Also if bindata was including the main,go, if you were to rebase or to merge directly without regenerating again the bindata, it would be incorrect. Let me know. FYI I will not have time for the next two weeks.

Reply:
I don't think this should be an issue unless another commit affected bindata.go. But you can rebase your branch onto master and force push your branch after fixing it. That should ensure there are no issues. IMO it's okay to rewrite the history of your feature branch.